### PR TITLE
[plg_system_languagefilter]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_system_languagefilter
### Testing Instructions

code review

### CODE REVIEW

File: plugins/system/languagefilter/languagefilter.xml [hunk 1]
```diff
@@ -19,76 +19,96 @@
...
 					default="default"
 					showon="item_associations:1[AND]alternate_meta:1[AND]xdefault:1"
-				>
+					>
 					<option value="default">PLG_SYSTEM_LANGUAGEFILTER_OPTION_DEFAULT_LANGUAGE</option>
 				</field>
-				<field name="remove_default_prefix" type="radio"
-					description="PLG_SYSTEM_LANGUAGEFILTER_FIELD_REMOVE_DEFAULT_PREFIX_DESC"
+
+				<field name="remove_default_prefix"
```
Comment (JoomliC) [inline plugins/system/languagefilter/languagefilter.xml]: Oups! you forgot there return line for name ;-)

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 9daac258f0daa2a4d67fa0ea64b66e7d8ccf3458<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10500">issues.joomla.org/joomla-cms/10500</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 9daac258f0daa2a4d67fa0ea64b66e7d8ccf3458<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10500">issues.joomla.org/joomla-cms/10500</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10500">issues.joomla.org/joomla-cms/10500</a>.</sub>

**Pull request merged**